packages/core/src/actions/execute.ts

```diff
@@ -92,12 +92,7 @@ export type Deployment = {
   treeSigners: Array<TreeSigner>
 }
 
-export type HandleError = (
-  e: any,
-  deployment: Deployment,
-  deploymentTransactionReceipts: ethers.TransactionReceipt[],
-  wallet?: ethers.Wallet
-) => Promise<void>
+export type HandleError = (e: any, deployment: Deployment) => Promise<void>
```

Comment:
I simplified many of the functions in the `deploymentContext` since verification now happens outside of the deployment process on the website. 

I also removed some other fields that are no longer required by the website for various reasons.